### PR TITLE
chore: update changelog

### PR DESCRIPTION
**Pull request by AAfghahi**
### SUMMARY

making sure 2.0 branch has a changelog that accurately reflects what's happening. 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,12 +18,21 @@ under the License.
 -->
 
 ## Change Log
+- [2.0.1](#201-fri-nov-4-103402-2022--0400)
+- [2.0.0](#200-tue-jun-28-085302-2022--0400)
```
Comment (eschutho) [inline CHANGELOG.md:22]: #20-tue-jun-28-085302-2022--0400

Comment (eschutho) [inline CHANGELOG.md:22]:
```suggestion
- [2.0.0](#20-tue-jun-28-085302-2022--0400)
```

File: UPDATING.md [hunk 2]
```diff
@@ -22,16 +22,33 @@ under the License.
 This file documents any backwards-incompatible changes in Superset and
 assists people when migrating to a new version.
 
-## 2.0.1
+## Next
```
Comment (eschutho) [inline UPDATING.md:25]: we shouldn't have a next in 2.0, but we'll have this in master. I think you'll want to create two different PRs.

Comment (AAfghahi) [reply UPDATING.md:25]: The one for master was already created and merged :)

Comment (AAfghahi) [reply UPDATING.md:25]: Ok already merged one with next into master.  Should I just deleted next here?

File: UPDATING.md [hunk 3]
```diff
@@ -24,14 +24,31 @@ assists people when migrating to a new version.
...
+
 ### Potential Downtime
 
+- [21284](https://github.com/apache/superset/pull/21284): A change which drops the unused `dbs.allow_multi_schema_metadata_fetch` column via a (potentially locking) DDL operation.
+
 ### Other
 
+## 2.0.1
+- [21895](https://github.com/apache/superset/pull/21895): Markdown components had their security increased by adhering to the same sanitization process enforced by Github. This means that some HTML elements found in markdowns are not allowed anymore due to the security risks they impose. If you're deploying Superset in a trusted environment and wish to use some of the blocked elements, then you can use the HTML_SANITIZATION_SCHEMA_EXTENSIONS configuration to extend the default sanitization schema. There's also the option to disable HTML sanitization using the HTML_SANITIZATION configuration but we do not recommend this approach because of the security risks. Given the provided configurations, we don't view the improved sanitization as a breaking change but as a security patch.
```
Comment (eschutho) [inline UPDATING.md:47]: this can go under other

File: UPDATING.md [hunk 4]
```diff
@@ -24,14 +24,31 @@ assists people when migrating to a new version.
 
 ## 2.0.1
 
-- [21895](https://github.com/apache/superset/pull/21895): Markdown components had their security increased by adhering to the same sanitization process enforced by Github. This means that some HTML elements found in markdowns are not allowed anymore due to the security risks they impose. If you're deploying Superset in a trusted environment and wish to use some of the blocked elements, then you can use the HTML_SANITIZATION_SCHEMA_EXTENSIONS configuration to extend the default sanitization schema. There's also the option to disable HTML sanitization using the HTML_SANITIZATION configuration but we do not recommend this approach because of the security risks. Given the provided configurations, we don't view the improved sanitization as a breaking change but as a security patch.
+- [22022](https://github.com/apache/superset/pull/22022): HTTP API endpoints `/superset/approve` and `/superset/request_access` have been deprecated and their HTTP methods were changed from GET to POST
```
Comment (eschutho) [inline UPDATING.md:27]: this whole 2.0.1 section should be deleted.

File: CHANGELOG.md [hunk 5]
```diff
@@ -18,12 +18,21 @@ under the License.
...
+- [1.4.2](#142-sat-mar-19-000806-2022-0200)
+- [1.4.1](#141)
+
 ### 2.0.1 (Fri Nov 4 10:34:02 2022 -0400)
 **Database Migrations**
 
 **Features**
 
 **Fixes**
+- [#22417](https://github.com/apache/superset/pull/22417) fix: fix: Force configuration for SafeMarkdown component in Handlebars(@geido)
```
Comment (eschutho) [inline CHANGELOG.md:35]:
```suggestion
- [#22417](https://github.com/apache/superset/pull/22417) fix: Force configuration for SafeMarkdown component in Handlebars(@geido)
```

**Review by eschutho — APPROVED** [review]

**Pull request merged**